src/test/java/de/tum/in/www1/artemis/TextExerciseIntegrationTest.java

```diff
@@ -18,13 +19,12 @@
 import de.tum.in.www1.artemis.domain.enumeration.Language;
 import de.tum.in.www1.artemis.domain.exam.ExerciseGroup;
 import de.tum.in.www1.artemis.domain.participation.Participation;
-import de.tum.in.www1.artemis.repository.ExampleSubmissionRepository;
-import de.tum.in.www1.artemis.repository.ExerciseRepository;
-import de.tum.in.www1.artemis.repository.TextExerciseRepository;
-import de.tum.in.www1.artemis.repository.TextSubmissionRepository;
+import de.tum.in.www1.artemis.repository.*;
```

Comment:
Codacy found an issue: [Avoid unused imports such as 'de.tum.in.www1.artemis.repository'](https://app.codacy.com/gh/ls1intum/Artemis/pullRequest?prid=5968855)

Reply:
See See f9e39e2dbe10ffa13483affe2de6359ac2244213